src/ucp/api/ucp.h

```diff
@@ -746,13 +746,16 @@ typedef struct ucp_worker_params {
     uint64_t                field_mask;
 
     /**
-     * Thread safety "mode" for the worker object and resources associated with it.
-     * This value is optional.
-     * If it's not set (along with its corresponding bit in the field_mask -
-     * UCP_WORKER_PARAM_FIELD_THREAD_MODE), the UCS_THREAD_MODE_SINGLE mode
-     * will be used. Note that actual thread mode may be different from mode
-     * passed to @ref ucp_worker_create. To get actual thread mode use
-     * @ref ucp_worker_query.
+     * The parameter thread_mode suggests the thread safety mode which worker
+     * and the associated resources should be created with. This is an
+     * optional parameter. The default value is UCS_THREAD_MODE_SINGLE and
+     * it is used when the value of the parameter is not set. When this
```

Comment:
I know that this wording was already approved for #3122, but in looking it over again, I have some suggested re-wording.  If this introduces undesirable inconsistencies, then we can stay with the existing wording.

Suggested re-wording plus adding references and emphasis for the parameter name:
The parameter `@a` thread_mode requests the thread safety mode with which the worker and the associated resources should be created. This is an optional parameter. The default value is `@ref` UCS_THREAD_MODE_SINGLE and is used when the value of the parameter is not set.

Reply:
@manjugv FYI